### DISCUSSION TITLE
I received a suspicious login email from Github support

### DISCUSSION THREAD

**Discussion by davik20** [community/community#65328] [New to GitHub] [answered]
Hi everyone,

  I  received a email from Github support about a suspicious login into my account a few hours ago. The email suggested I reset my password and ensure more security measures like 2 factor authentication.

I logged into my Github account and checked the security logs and saw updates by a "github-staff" account destroying oauth to all the applications i have logged into.

![Screenshot 2023-08-29 at 11 42 58](https://github.com/community/community/assets/50995003/85c753a9-68e3-4e2e-86aa-b9c98f335366) 

a few minutes ago. I received the same email about a suspicious login and saw that github-staff account had made some more suspicious updates to oauth

What could be going. This doesn't look normal to me, could this be a hack?

**Comment by Storm99999** [comment]
same here.

**Comment by ettaboyle** [comment, marked as answer]
Hi there, thanks for sharing with us. This protocol is normal and is triggered in the event of an user account security concern. If you haven't already, please use our [Reinstatement Form](https://support.github.com/contact/reinstatement) to recover your account and our team will get you sorted.

**Comment by aamaanaa** [comment]
i had the very same. reseted password. all auths got removed.

other than my usual locations, no others accessed my account
